### DISCUSSION TITLE
Point Elevation/Altitude

### DISCUSSION THREAD

**Discussion by mikeroh** [osmandapp/OsmAnd#14535] [Ideas]
When out hiking or skiing, I frequently find myself wanting to know the altitude of a particular point on the map. It is not always fast or easy to count the contour lines. 

When I long press any point on the map, it would be great if the elevation of this point could be presented to the user on either of the two screens shown below. Another idea would be to show the elevation difference between this point and my current location next to the distance/bearing icon.

Is this something others would want? I'm not sure I have the coding skills to add this, but I might start poking around. If anybody else could add this feature, or show me where I could add this feature, that would be great!

Thanks,
Michael

![image](https://user-images.githubusercontent.com/20286710/173452036-567cde08-c779-40ba-85c2-2442eb9650cd.png)
![image](https://user-images.githubusercontent.com/20286710/173452048-6f4d4b81-f352-4167-85f8-4a1406346466.png)

**Comment by scaidermern** [comment]
Duplicate of #2487.

**Reply by mikeroh** [reply]
Ah thanks, It seems like my search was only for discussions, not issues so I didn't find #2487. Sorry about that.
